tests/tests_manifests/test_manifest.py

```diff
@@ -34,12 +34,21 @@ def test_manifest_is_abstract(self):
         )
 
     def test_invalid_version(self):
```

Comment:
Split this unit test into multiple individual tests for each invalid-schema-version- scenario.

Reply:
Sure @dblock I will make these changes. Thank you for your input.